src/enclave_proc/resource_manager.rs

```diff
@@ -1087,6 +1089,19 @@ fn release_enclave_descriptor(enc_fd: RawFd) -> NitroCliResult<()> {
     Ok(())
 }
 
+/// Computes the ceil of `lhs / rhs`.
```

Comment:
Can we place this function in utils ?

Reply:
Placed it for the moment in utils.rs, however at the moment that file contains things mainly related to the console functionality. Maybe in a future update we will split the file in a proper `utils.rs` and a `console.rs`?